apps/mull-ui/src/constants.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Map of the current routes for the application
+ */
+export const ROUTES = {
```

Comment:
If they are constants and the values in the properties don't change, might as well use enums from TypeScript.

Reply:
Sure why not